### PR TITLE
Menu bar

### PR DESCRIPTION
**Pull request by bodik4real**
added menu bar and fixed issues related to login

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -3,14 +3,46 @@ import { CommonModule } from '@angular/common';
...
+    path: 'taskList',
+    component: TaskListComponent,
+    canActivate: [AuthGuard]
+  },
+  {
+    path: '',
+    component: LoginComponent,
+    pathMatch: 'full',      
+  },
+ /* {
```
Comment (timonsk) [inline src/app/app-routing.module.ts]: PLease remove commented code

File: src/app/app.component.html [hunk 2]
```diff
@@ -1,4 +1,2 @@
+<app-menu-bar *ngIf="isLogining()"></app-menu-bar>
```
Comment (timonsk) [inline src/app/app.component.html]: Please rename to IsAuthenticated

File: src/app/app.component.ts [hunk 3]
```diff
@@ -8,4 +8,10 @@ import {Router} from '@angular/router';
 })
 export class AppComponent {
   title = 'ToDoListApp';
+
+  constructor(private userDataService: UserDataService) {}
+
+  public isLogining():boolean{
+    return this.userDataService.isLogining();
```
Comment (timonsk) [inline src/app/app.component.ts]: Please use here the same method as you are using in AuthGuard to check whether a user is authenticated

File: src/app/app.module.ts [hunk 4]
```diff
@@ -26,11 +27,12 @@ import { PageNotFoundComponent } from './page-not-found/page-not-found.component
   declarations: [
     AppComponent,
     PageNotFoundComponent,
+    MenuBarComponent
```
Comment (timonsk) [inline src/app/app.module.ts]: Let's rename MenuBarComponent to MenuComponent

File: src/app/login/login/login.component.html [hunk 5]
```diff
@@ -1,13 +1,9 @@
...
-<script src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.min.js"></script>
-<script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
-
 <div class="container input-group">
   <div class="row">
     <div class="col-sm-6 col-md-4 col-md-offset-4">
       <h1 class="text-center login-title">Sign in to Your task list</h1>
       <div class="account-wall">
-        <img class="profile-img" src="./../../../assets/images/photo.png"
+        <img class="profile-img" src="../../../assets/Images/photo.png"
```
Comment (timonsk) [inline src/app/login/login/login.component.html]: Please rename folder to image the do the same with a real folder in assets

File: src/app/menu-bar/menu-bar.component.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```
Comment (timonsk) [inline src/app/menu-bar/menu-bar.component.html]: Do we really need it here ??? Did you know that we use <html lang="en"> only one per app, we can use it also only in iframe

File: src/app/menu-bar/menu-bar.component.html [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+<html lang="en">
+
+<head>
+  <title>TaskManager</title>
+  <meta charset="utf-8">
+</head>
+
+<body>
+  <nav class="navbar navbar-inverse">
```
Comment (timonsk) [inline src/app/menu-bar/menu-bar.component.html]: Please remove all redundant code here and leave the only navigation

File: src/app/menu-bar/menu-bar.component.html [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="container-fluid">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="#">TaskManager</a>
+      </div>
+      <ul class="nav navbar-nav">
+        <li class="active">
+          <a href="#">Home</a>
+        </li>
+        <li class="dropdown">
+          <a class="dropdown-toggle" data-toggle="dropdown" href="#">Home
```
Comment (timonsk) [inline src/app/menu-bar/menu-bar.component.html]: Do we really have such menu items?

File: src/app/menu-bar/menu-bar.component.html [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+            <li>
+              <a href="#">Func2</a>
+            </li>
+            <li>
+              <a href="#">Func3</a>
+            </li>
+          </ul>
+        </li>
+        <li>
+          <a href="#">MyList</a>
```
Comment (timonsk) [inline src/app/menu-bar/menu-bar.component.html]: Please create appropriate menu items as we discussed

File: src/app/services/user-data.service.ts [hunk 10]
```diff
@@ -70,4 +68,12 @@ export class UserDataService {
   public saveUser(user: User) {
     localStorage.setItem(this.currentUserKey, JSON.stringify(user));
   }
+
+  public isLogining(): boolean {
```
Comment (timonsk) [inline src/app/services/user-data.service.ts]: Please remove this method

File: src/app/menu-bar/menu-bar.component.html (hunk 7 above)
Comment (bodik4real) [reply src/app/menu-bar/menu-bar.component.html]: all  redundant  code is removed.

File: src/app/menu-bar/menu-bar.component.html (hunk 6 above)
Comment (bodik4real) [reply src/app/menu-bar/menu-bar.component.html]: ok, it’s already removed

File: src/app/services/user-data.service.ts (hunk 10 above)
Comment (bodik4real) [reply src/app/services/user-data.service.ts]: method is removed

File: src/app/menu-bar/menu-bar.component.html (hunk 9 above)
Comment (bodik4real) [reply src/app/menu-bar/menu-bar.component.html]: Created

File: src/app/login/login/login.component.html (hunk 5 above)
Comment (bodik4real) [reply src/app/login/login/login.component.html]: it is already renamed

File: src/app/app.module.ts (hunk 4 above)
Comment (bodik4real) [reply src/app/app.module.ts]: renamed

File: src/app/app.component.ts (hunk 3 above)
Comment (bodik4real) [reply src/app/app.component.ts]: used

File: src/app/app.component.html (hunk 2 above)
Comment (bodik4real) [reply src/app/app.component.html]: renamed

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (bodik4real) [reply src/app/app-routing.module.ts]: removed

**Pull request merged**